src/extension.ts

```diff
@@ -86,6 +86,11 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
       coursierPath,
       "fetch",
       "-p",
+      "--ttl",
+      // Use infinite ttl to avoid redunant "Checking..." logs when using SNAPSHOT
+      // versions. Metals SNAPSHOT releases are effectively immutable since we
+      // never publish the same version twice.
+      "Inf",
```

Comment:
I think this breaks the `publishLocal` workflow though

Reply:
I mean, I can live with that, it's just a matter of deleting the artifact locally before re-launching the server, but I would document this.

Reply:
I don't think it breaks the publish local flow since `publishLocal` overrides the jars themselves. The default ttl is 24hr so you would have already noticed if it breaks `publishLocal`

Reply:
Got it. LGTM then